Sidney-/slugtrade
# Add closet item refactor

New Functionality:
- Updated method of adding photos to closet items
- Increased the width of the add closet items panel (small css change)

How to test:
- Add and delete a whole bunch of items in a whole bunch of different ways!!
- Make sure these properties are always upheld:
  - Items with the ? are always on the far right side of the container (never in the middle or left)
  - You cannot add another photo selector without first selecting images for all of your current file selectors
  - You can submit variable amounts of images (after adding and deleting a bunch)
  - When you try to submit without adding at least 1 photo, you get an alert
  - Images always appear in the middle of the containers and scale in a way that makes sense

Notes:
- There are a lot of weird edge cases that I found while testing myself, which is why it took me so long to create this. Please be thorough in your testing. I wouldn't be surprised if you find something I overlooked. Thank you!!
